README.md

```diff
@@ -1,7 +1,6 @@
 # Horreum
 
-Horreum is a service for storing performance data and regression analysis.
-
+Horreum is a service for storing performance data and regression analysis. For more information please visit our 
```

Comment:
Separate sentences with a newline.

"
Please, visit our project website

  <URL>

for more information.
"